### DISCUSSION TITLE
What's the mainGpu parameter do?

### DISCUSSION THREAD

**Discussion by ramz35** [Mobile-Artificial-Intelligence/maid#403] [Q&A]
I see that for local models the mainGPU parameter is available (Y) - what does this do?

What is the way to get maid working with GPU directly with local models?
I assume if Ollama is used then GPU is handled via that?

When I run maid I do get a 10% spike in GPU but CPU goes to 50-60% so I'm not sure what's going on and what the program is trying to use (obviously some CPU) not where to tune the settings.

I'm Windows NVidia.

**Comment by danemadsen** [comment]
Yeah when using the Ollama API gpu is managed by ollama. The mainGPU param just specifies which GPU ollama should use
